configure.py

```diff
@@ -326,6 +326,7 @@ def find_headers(repodir, excluded_dirs):
     'test/boost/linearizing_input_stream_test',
     'test/boost/loading_cache_test',
     'test/boost/log_heap_test',
+    'test/boost/estimated_histogram_test',
     'test/boost/logalloc_test',
```

Comment:
You can merge this with the previous patch.

Reply:
ok